pizza_cutter/des_pizza_cutter/_coadd_slices.py

```diff
@@ -138,12 +135,6 @@ def _build_slice_inputs(
         any pixels with any of the noise_interp_flags. It is the fraction of
         the subset of the SE image that approximatly overlaps the final coadded
         region.
-    max_unmasked_trail_fraction : float
```

Comment:
how do you know the trail exists but was unmasked?

Reply:
This cut never really worked which is why it is being removed.